spyder/config/utils.py

```diff
@@ -50,6 +50,28 @@
     (_("Markdown files"), ('.md', )),
 ]
 
+
+LANGUAGE_EXTENSIONS = {
```

Comment:
If you're not planning to use this dictionary in the SimpleCodeEditor, please remove it.

Else, please add a TODO comment above it to understand why you're putting it here and not using it.

Reply:
Forgot to remove it. Fixed.  (tracking this TODO on a separate issue already)